lib/Net/HTTP/Methods.pm

```diff
@@ -638,3 +638,7 @@ sub inflate_ok {
 } # BEGIN
 
 1;
+
+__END__
+
+# ABSTRACT: Represents method for Net::HTTP
```

Comment:
Maybe we could change this to:

`# ABSTRACT: Methods shared by Net::HTTP and Net::HTTPS

What do you think?